packages/css/field.css

```diff
@@ -1,7 +1,67 @@
 .ds-field {
-  display: contents;
+  align-items: start;
+  display: flex;
+  flex-direction: column;
+  gap: var(--ds-spacing-2);
 
-  & > * + * {
-    margin-top: var(--ds-spacing-2);
+  @composes ds-body-text--md from './base/base.css';
+
+  &[data-size='sm'] {
+    @composes ds-body-text--sm from './base/base.css';
+  }
+
+  &[data-size='lg'] {
+    @composes ds-body-text--lg from './base/base.css';
+  }
+
+  & [data-field='description'] {
+    color: var(--ds-color-neutral-text-subtle); /* TODO: Change to opacity or color-mix(currentColor, trasparent) to ensure contrast when parent element color changes? */
+  }
+
+  /**
+   * States
+   */
+  &:has([aria-disabled='true'], :disabled) > * {
+    cursor: not-allowed;
+    opacity: var(--ds-disabled-opacity);
+  }
+
+  &:has([aria-readonly='true'], [readonly]) label {
+    --dsc-label--readonly: ; /* Activate lock icon for label when readonly */
+  }
+
+  /**
+   * Toggle inputs
+   */
+  &:has(input:is([type='radio'], [type='checkbox'])) {
+    border-radius: var(--ds-border-radius-md);
+    display: grid;
+    grid-template-columns: auto 1fr;
```

Comment:
`grid-template-columns` and `focus` on checkboxes with `aria-label/describedby `🙄.
 
![image](https://github.com/user-attachments/assets/ac9bd519-e161-45f4-8f23-9cd5e39560ec)

Reply:
Oh! very nice catch! Will look into it

Reply:
Fixed ✅